finance/bitcoin/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem              1.0
 
 name                    bitcoin
 categories              finance crypto
-version                 0.14.1
+version                 0.15.1
 revision                1
```

Comment:
When increasing the version, the revision line should be set to 0. Since that is the default value, just remove the revision line entirely.